app/service/data_svc.py

```diff
@@ -86,6 +86,15 @@ async def locate(self, object_name, match=None):
         except Exception as e:
             self.log.error('[!] LOCATE: %s' % e)
 
+    async def search(self, value, object_name=None):
+        try:
+            if object_name:
+                return [obj for obj in self.ram[object_name] if obj.search_tags(value)]
+            else:
```

Comment:
should we allow for cross object searching, or should it always require an object type?  I'm thinking it might be nice for display functionality in a UI, but I can't think of an operational need.

Reply:
I think it may cause more good than bad. let's start with just a specified object_name